stocks/historical.py

```diff
@@ -0,0 +1,28 @@
+"""This module is for loading historical data for stocks"""
+from yahoo_historical import Fetcher
+
+import arrow
+from .models import Stock, DailyStockQuote
+
+def loadData10YearsBackFromTodayForStock(request):
+    """Function that creates Stock object and loads data for it 10 years back"""
+    if request.method == "POST":
+        body = request.POST
+        name = body['name']
+        ticker = body['ticker']
+        stock = Stock(name=name, ticker=ticker)
+        stock.save()
+        now = arrow.now()
+        tenBack = now.replace(years=-10)
+        now = getDateArrayForFetcher(now)
+        tenBack = getDateArrayForFetcher(tenBack)
+        fetcher = Fetcher(ticker, tenBack, now)
+        history = fetcher.getHistorical()
+        for row in history.itertuples():
+            value = getattr(row, "Close")
+            date = getattr(row, "Date")
+            DailyStockQuote(value=value, date=date, stock=stock).save()
```

Comment:
Maybe do a bulk update `DailyStockQuote. objects.bulk_create([DailyStockQuote(value=value, date=date, stock=stock),...])`

Reply:
What are the max entries for bulk create?